distribution/openhabhome/conf/services/cometvisu.cfg

```diff
@@ -0,0 +1,21 @@
+cometvisu:webFolder=../../../CometVisu/src/
```

Comment:
The config should actually NOT have the cometvisu: prefix - this is automatically derived from the filename.